soapui-installer/src/install4j/SoapUI.install4j

```diff
@@ -802,7 +802,8 @@ if (Util.isMacOS()) {
                     <element index="10">-Djava.library.path=${installer:rootInstallDir}/bin</element>
                     <element index="11">-Djava.util.Arrays.useLegacyMergeSort=true</element>
                     <element index="12">-splash:SoapUI-Spashscreen.png</element>
-                    <element index="13">-Dlog4j2.formatMsgNoLookups=true</element>
+                    <element index="13">--illegal-access=permit</element>
+                    <element index="14">-Dlog4j2.formatMsgNoLookups=true</element>

```

Comment:
Change the value of the attribute "length" for parent "property" element, since you have added a new vmoption.

Reply:
Thx, have no IDE, and didn't know about it.